locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml

```diff
@@ -150,108 +144,102 @@ de:
         sequence-29:
           s: 29
           from: 00:02:49,13
           to: 00:02:55,10
-          text: My brother had been going to high school at a boarding school in Parma.
+          text: Mein Bruder besuchte die Mittelstufe auf einem Internat in Parma.
         sequence-30:
           s: 30
           from: 00:02:55,10
           to: 00:02:59,27
-          text: There was a need to move closer to the city.
+          text: Darum war es nötig dichter an die Stadt zu ziehen.
         sequence-31:
           s: 31
           from: 00:02:59,27
           to: 00:03:03,01
-          text: They bought a house in Dalmazia street.
+          text: Sie kauften ein Haus in der Dalmazia Straße.
         sequence-32:
           s: 32
           from: 00:03:03,01
           to: 00:03:07,03
-          text: Life changed dramatically for me.
+          text: Mein Leben veränderte sich schlagartig.
         sequence-33:
           s: 33
           from: 00:03:07,03
           to: 00:03:19,09
-          text: We left farm life, based on conservatism, traditionalism, on having
-            to be careful to what you say or do.
+          text: Wir verließen das konservative, traditionelle Landleben, wo man vorsichtig sein musste, was man sagte.
         sequence-34:
           s: 34
           from: 00:03:19,09
           to: 00:03:25,23
-          text: Also, women would be considered being a nobody.
+          text: Auch waren Frauen dort nichts wert.
         sequence-35:
           s: 35
           from: 00:03:25,23
           to: 00:03:30,14
-          text: I used to be told to shut up because I was a woman even at home.
+          text: Ich sollte immer den Mund halten, weil ich eine Frau war, sogar zuhause.
         sequence-36:
           s: 36
           from: 00:03:30,14
           to: 00:03:34,02
-          text: My mother thought this was right.
+          text: Meine Mutter hielt das für richtig.
         sequence-37:
           s: 37
           from: 00:03:34,02
           to: 00:03:38,07
-          text: In the city I started meeting women who went to work.
+          text: In der Stadt traf ich zum ersten mal auf Frauen, die arbeiten gingen.
         sequence-38:
           s: 38
           from: 00:03:38,07
           to: 00:03:45,11
-          text: The working class elite used to live in Dalmazia street.
+          text: Die Elite der Arbeiterklasse lebte damals in der Dalmazia Straße.
         sequence-39:
           s: 39
           from: 00:03:45,11
           to: 00:03:50,03
-          text: Reggiane factory technicians of the water and gas companies.
+          text: Reggio Fabriktechniker der Wasser- und Gasfirmen.
         sequence-40:
           s: 40
           from: 00:03:50,03
           to: 00:03:57,12
-          text: They were a very bright, mature and well-prepared group of working
-            class people.
+          text: Es war sehr intelligente, reife und gut vorbereitete Gruppe von Leuten der Arbeiterklasse.
```

Comment:
I'm not really sure about this sentence... Maybe someone with Italian knowledge can help out here?
@busfaktor2 @ERA-aishah @eabalos 
"c’è un tipo di classe operaia molto consapevole, molto matura, molto preparata...", see http://www.resistance-archive.org/downloads/transcripts/anita-malavasi-it.pdf